chia/wallet/wallet_nft_store.py

```diff
@@ -272,3 +272,8 @@ async def rollback_to_block(self, height: int) -> bool:
             if result.rowcount > 0:
                 return True
             return False
+
+    async def delete_wallet(self, wallet_id: uint32) -> None:
```

Comment:
why not put this in `nft_wallet`, wallet_nft_store is store for NFTs?

Reply:
Its like with all other "stores" we have. When something gets changed inside the store, the method is inside since it accessed the DB of the store and this method is supposed to remove entries from the store so i guess it should be in there rather than `NFTWallet`, no?